src/lib/components/primitives/alert/alert.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
```

Comment:
This is the only new component being added. This needs some design love, but leaving as is to keep moving forward.

Reply:
Looks good so far